src/Components/Aspire.Qdrant/README.md

```diff
@@ -0,0 +1,107 @@
+# Aspire.Qdrant.Client library
```

Comment:
Can you also add entries for this component in:

* https://github.com/dotnet/aspire/blob/main/src/Components/Aspire_Components_Progress.md
* https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md